Shared/src/Datahub.Shared/Entities/TerraformTemplate.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Datahub.Shared.Entities;
 
-public class TerraformTemplate(string name, string status)
+public class TerraformTemplate(string name, string status, DateTime requestedAt)
```

Comment:
don't we have DateTime.UtcNow as only use case for requestAt? Any reason we don't use a DateTimeOffset?

Reply:
i updated this